todolist/settings.py

```diff
@@ -23,7 +23,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
The `DEBUG` setting is set to `True`. This should be set to `False` in a production environment to prevent the display of sensitive information.

Reply:
Don't give me a shit like this! It is a leaning project with clear task. And this DEBUG thing is overload. Forget this , it is not important now!